biospecimen-tracking/src/main/resources/PhenoTips/UIX_Section__biospecimen.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
See comment from previous commit -- this should be `UIX_Section__biospecimens`, with an `s` at the end, in order to be consistent with the field.